<!DOCTYPE html>
<html><head><title>DataLakeAclChangeFailedException</title><link rel="stylesheet" href="../../styles.css"/><script src="../../scripts.js"></script></head><body onload="ro();">
<div class="rH">2 instantiations of DataLakeAclChangeFailedException</div><div class="rA">Azure.Storage.Files.DataLake (2)</div><div class="rG" id="Azure.Storage.Files.DataLake"><div class="rF"><div class="rN">DataLakeErrors.cs (2)</div>
<a href="../DataLakeErrors.cs.html#44"><b>44</b>=&gt; new <i>DataLakeAclChangeFailedException</i>(</a>
<a href="../DataLakeErrors.cs.html#56"><b>56</b>=&gt; new <i>DataLakeAclChangeFailedException</i>(</a>
</div>
</div>
<div class="rH">28 references to DataLakeAclChangeFailedException</div><div class="rA">Azure.Storage.Files.DataLake (13)</div><div class="rG" id="Azure.Storage.Files.DataLake"><div class="rF"><div class="rN">DataLakeErrors.cs (4)</div>
<a href="../DataLakeErrors.cs.html#43"><b>43</b>public static <i>DataLakeAclChangeFailedException</i> ChangeAclRequestFailed(RequestFailedException exception, string continuationToken)</a>
<a href="../DataLakeErrors.cs.html#50"><b>50</b>$&quot;{nameof(<i>DataLakeAclChangeFailedException</i>.ContinuationToken)}={continuationToken} &quot; +</a>
<a href="../DataLakeErrors.cs.html#55"><b>55</b>public static <i>DataLakeAclChangeFailedException</i> ChangeAclFailed(Exception exception, string continuationToken)</a>
<a href="../DataLakeErrors.cs.html#59"><b>59</b>$&quot;{nameof(<i>DataLakeAclChangeFailedException</i>.ContinuationToken)}={continuationToken} after addressing the error.&quot;,</a>
</div>
<div class="rF"><div class="rN">DataLakePathClient.cs (7)</div>
<a href="../DataLakePathClient.cs.html#2334"><b>2334</b>/// A &lt;see cref=&quot;<i>DataLakeAclChangeFailedException</i>&quot;/&gt; will be thrown if a request to</a>
<a href="../DataLakePathClient.cs.html#2381"><b>2381</b>/// A &lt;see cref=&quot;<i>DataLakeAclChangeFailedException</i>&quot;/&gt; will be thrown if a request to</a>
<a href="../DataLakePathClient.cs.html#2428"><b>2428</b>/// A &lt;see cref=&quot;<i>DataLakeAclChangeFailedException</i>&quot;/&gt; will be thrown if a request to</a>
<a href="../DataLakePathClient.cs.html#2475"><b>2475</b>/// A &lt;see cref=&quot;<i>DataLakeAclChangeFailedException</i>&quot;/&gt; will be thrown if a request to</a>
<a href="../DataLakePathClient.cs.html#2522"><b>2522</b>/// A &lt;see cref=&quot;<i>DataLakeAclChangeFailedException</i>&quot;/&gt; will be thrown if a request to</a>
<a href="../DataLakePathClient.cs.html#2569"><b>2569</b>/// A &lt;see cref=&quot;<i>DataLakeAclChangeFailedException</i>&quot;/&gt; will be thrown if a request to</a>
<a href="../DataLakePathClient.cs.html#2628"><b>2628</b>/// A &lt;see cref=&quot;<i>DataLakeAclChangeFailedException</i>&quot;/&gt; will be thrown if a request to</a>
</div>
<div class="rF"><div class="rN">Models\DataLakeAclChangeFailedException.cs (2)</div>
<a href="../Models/DataLakeAclChangeFailedException.cs.html#22"><b>22</b>/// &lt;summary&gt;Initializes a new instance of the &lt;see cref=&quot;<i>DataLakeAclChangeFailedException</i>&quot;&gt;&lt;/see&gt; class with a specified error message, HTTP status code, error code, and a reference to the inner exception that is the cause of this exception.&lt;/summary&gt;</a>
<a href="../Models/DataLakeAclChangeFailedException.cs.html#35"><b>35</b>/// &lt;summary&gt;Initializes a new instance of the &lt;see cref=&quot;<i>DataLakeAclChangeFailedException</i>&quot;&gt;&lt;/see&gt; class with a specified error message, HTTP status code, error code, and a reference to the inner exception that is the cause of this exception.&lt;/summary&gt;</a>
</div>
</div>
<div class="rA">Azure.Storage.Files.DataLake.Tests (15)</div><div class="rG" id="Azure.Storage.Files.DataLake.Tests"><div class="rF"><div class="rN">DirectoryClientTests.cs (15)</div>
<a href="../../Azure.Storage.Files.DataLake.Tests/DirectoryClientTests.cs.html#1232"><b>1232</b>catch (<i>DataLakeAclChangeFailedException</i>)</a>
<a href="../../Azure.Storage.Files.DataLake.Tests/DirectoryClientTests.cs.html#1563"><b>1563</b>catch (<i>DataLakeAclChangeFailedException</i>)</a>
<a href="../../Azure.Storage.Files.DataLake.Tests/DirectoryClientTests.cs.html#1779"><b>1779</b>await TestHelper.AssertExpectedExceptionAsync&lt;<i>DataLakeAclChangeFailedException</i>&gt;(</a>
<a href="../../Azure.Storage.Files.DataLake.Tests/DirectoryClientTests.cs.html#1809"><b>1809</b>await TestHelper.AssertExpectedExceptionAsync&lt;<i>DataLakeAclChangeFailedException</i>&gt;(</a>
<a href="../../Azure.Storage.Files.DataLake.Tests/DirectoryClientTests.cs.html#1834"><b>1834</b>await TestHelper.AssertExpectedExceptionAsync&lt;<i>DataLakeAclChangeFailedException</i>&gt;(</a>
<a href="../../Azure.Storage.Files.DataLake.Tests/DirectoryClientTests.cs.html#1932"><b>1932</b>catch (<i>DataLakeAclChangeFailedException</i>)</a>
<a href="../../Azure.Storage.Files.DataLake.Tests/DirectoryClientTests.cs.html#2271"><b>2271</b>catch (<i>DataLakeAclChangeFailedException</i>)</a>
<a href="../../Azure.Storage.Files.DataLake.Tests/DirectoryClientTests.cs.html#2487"><b>2487</b>await TestHelper.AssertExpectedExceptionAsync&lt;<i>DataLakeAclChangeFailedException</i>&gt;(</a>
<a href="../../Azure.Storage.Files.DataLake.Tests/DirectoryClientTests.cs.html#2517"><b>2517</b>await TestHelper.AssertExpectedExceptionAsync&lt;<i>DataLakeAclChangeFailedException</i>&gt;(</a>
<a href="../../Azure.Storage.Files.DataLake.Tests/DirectoryClientTests.cs.html#2542"><b>2542</b>await TestHelper.AssertExpectedExceptionAsync&lt;<i>DataLakeAclChangeFailedException</i>&gt;(</a>
<a href="../../Azure.Storage.Files.DataLake.Tests/DirectoryClientTests.cs.html#2641"><b>2641</b>catch (<i>DataLakeAclChangeFailedException</i>)</a>
<a href="../../Azure.Storage.Files.DataLake.Tests/DirectoryClientTests.cs.html#2974"><b>2974</b>catch (<i>DataLakeAclChangeFailedException</i>)</a>
<a href="../../Azure.Storage.Files.DataLake.Tests/DirectoryClientTests.cs.html#3190"><b>3190</b>await TestHelper.AssertExpectedExceptionAsync&lt;<i>DataLakeAclChangeFailedException</i>&gt;(</a>
<a href="../../Azure.Storage.Files.DataLake.Tests/DirectoryClientTests.cs.html#3220"><b>3220</b>await TestHelper.AssertExpectedExceptionAsync&lt;<i>DataLakeAclChangeFailedException</i>&gt;(</a>
<a href="../../Azure.Storage.Files.DataLake.Tests/DirectoryClientTests.cs.html#3245"><b>3245</b>await TestHelper.AssertExpectedExceptionAsync&lt;<i>DataLakeAclChangeFailedException</i>&gt;(</a>
</div>
</div>
</body></html>